aspnetcore/index.md

```diff
@@ -87,6 +89,27 @@ To change the sample to run the `ExpandDefault` scenario, define the `ExpandDefa
 
 For more information on using [C# preprocessor directives](/dotnet/csharp/language-reference/preprocessor-directives/) to selectively compile sections of code, see [#define (C# Reference)](/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-define) and [#if (C# Reference)](/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if).
 
+### Sample regions
+
+Documentation sample apps contain sections of code surrounded by [#region](/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-region) and [#end-region](/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-endregion) C# statements. The documentation build system injects these regions into the rendered documentation topics.  
+
+Region names usually contain the word "snippet." The following example shows a region named `snippet_FilterInCode`:
```

Comment:
After this, show the line of code in markdown that pulls in this snippet.

Reply:
I didn't do it for two reasons:

* It blows our 85-ish char limit to show it.
* This section is just about why they're there. If they want to know more, the linked content takes care of it.

I'll put in a *fake* markdown line (shorter) to show it ... try to keep it <85 chars.